### DISCUSSION TITLE
import html saved by SingleFile to OneNote

### DISCUSSION THREAD

**Discussion by shawn8888** [gildas-lormeau/SingleFile#1094] [Q&A]
I have a lot of HTML files saved by SingleFile. How can I import it into OneNote to make them searchable?
I have tried the OneNote Clip browser extension. None of the options can save the pictures on HTML files.

**Comment by gildas-lormeau** [comment]
If I remember correctly, OneNote does not support HTML files.

**Reply by shawn8888** [reply]
> If I remember correctly, OneNote does not support HTML files.

That's true. 
OneNote clip addon cannot clip articles from HTML files opened in Firefox or other browser.
